app/_src/kubernetes-ingress-controller/references/prometheus.md

```diff
@@ -17,29 +17,35 @@ This document is a reference for the former type.
 
 This metric provides the following labels:
 
-* `protocol` describes the configuration protocol in use, which can be `db-less` or `deck`. 
+* `protocol` describes the configuration protocol in use, which can be `db-less` or `deck`.
+{% if_version gte:2.9.x %}
+* `dataplane` describes the dataplane that was the target of configuration push.
```

Comment:
```suggestion
* `dataplane` describes the data plane that was the target of configuration push.
```
Data plane, two words, is our standard term (though you'll find a lot of instances of both dataplane and data plane in the docs, from before we standardized on just one).

Mesh/Kuma is the exception, where they specifically use "dataplane" to refer to the entity.